src/renderer/components/ft-video-player/ft-video-player.js

```diff
@@ -229,7 +233,11 @@ export default Vue.extend({
         this.player.on('mouseleave', this.removeMouseTimeout)
 
         this.player.on('volumechange', this.updateVolume)
-        this.player.controlBar.getChild('volumePanel').on('mousewheel', this.mouseScrollVolume)
+        if (this.videoVolumeMouseScroll) {
```

Comment:
I would recommend using the on `wheel` event rather than the `mousewheel` event due to `mousewheel` being [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/Element/mousewheel_event). 
This would only really matter if the application transitions to a web native app rather than an electron app, but I figured it may be worth pointing out.

Reply:
Could u place this in #1284?